.eslintrc.yml

```diff
@@ -8,3 +8,5 @@ rules:
   indent: [error, 2, { MemberExpression: "off", SwitchCase: 1 }]
   no-trailing-spaces: error
   no-unused-vars: [error, { vars: all, args: none, ignoreRestSiblings: true }]
+parserOptions:
+  ecmaVersion: 2018
```

Comment:
is this change needed still? i think it was needed in an earlier revision but no longer so?

Reply:
You are right, thanks! I'll make all the updates and then push the correct version :)